server/src/main/java/org/opensearch/cluster/coordination/CoordinationMetadata.java

```diff
@@ -149,6 +150,16 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeCollection(votingConfigExclusions);
     }
 
+    public void writeToSorted(StreamOutput out) throws IOException {
+        out.writeLong(term);
+        lastCommittedConfiguration.writeToSorted(out);
+        lastAcceptedConfiguration.writeToSorted(out);
+        List<VotingConfigExclusion> sortedList = votingConfigExclusions.stream()
+            .sorted(Comparator.comparing(VotingConfigExclusion::getNodeId))
+            .collect(Collectors.toList());
+        out.writeCollection(sortedList);
+    }
+
```

Comment:
Should we instead have a method in StreamOutput like writeSortedCollection to foster code re-usability and minimising custom logic